plugins/modules/certificate.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: certificate
+
+author:
+  - Domen Dobnikar (@domen_dobnikar)
+short_description: Handles cluster SSL certificates.
+description:
+  - Can upload and verify SSL certificates.
+version_added: 1.1.0
+extends_documentation_fragment:
+  - scale_computing.hypercore.cluster_instance
+seealso: []
+options:
+  certificate:
+    description:
+      - File location containing the X.509 PEM-encode certificate.
+    type: str
+    required: true
+  private_key:
+    description:
+      - File location containing the RSA PEM-encoded private key.
+    type: str
+    required: true
+notes:
+  - C(check_mode) is not supported.
+"""
+
+EXAMPLES = r"""
+- name: Upload new certificate
+  scale_computing.hypercore.certificate:
+    private_key: "{{ lookup('file', 'private_key.pem') }}"
+    certificate: "{{ lookup('file', 'scale_cert.cer') }}"
+"""
+
+RETURN = r"""
+record:
+  description:
+    - Certificate record.
+  returned: success
+  type: dict
+  sample:
+    certificate: this_is_the_certificate
```

Comment:
Change to
```
  sample:
    certificate: |
      -----BEGIN CERTIFICATE-----
      MIIGKzCCBBOgAwIBAgIUWaGzXfgSUuwwPJu3F2Q/Ru/O8JQwDQYJKoZIhvcNAQEL
      ...
      -----END CERTIFICATE-----
```

I never know that is the format of certificate (pem, der, p12, etc).

Reply:
I will add this to give more info. We do already have this info in the parameter description. 
` - File location containing the X.509 PEM-encode certificate.`